_i18n/en/documentation/general/ads.md

```diff
@@ -6,14 +6,14 @@ If you are seeing ads, this can have two reasons:
 
 AntennaPod is open-source. This means that anyone can download the source code. Unfortunately, some developers take AntennaPod, insert ads and then publish it as a new app. You can get the official AntennaPod app on [Google Play](https://play.google.com/store/apps/details?id=de.danoeh.antennapod) and [F-Droid](https://f-droid.org/packages/de.danoeh.antennapod/). You can even get the latest release from [GitHub](https://github.com/AntennaPod/AntennaPod/) and build AntennaPod yourself!
 
-### 2. The podcast publishers added the ads to their shownotes or audio file
+### 2. The podcast creators or their hosting company added the ads to their shownotes or audio file
 
-AntennaPod plays podcasts just like they were published by the creators. If they include ads, AntennaPod plays them. We have nothing to do with those ads, so the revenue fully goes to the publishers of your podcast.
+AntennaPod plays podcasts just like they were published by the creators. If they include ads, AntennaPod plays them. We have nothing to do with those ads, so all revenues go to the podcast creators.
```

Comment:
```suggestion
AntennaPod plays podcasts just like they were published by the creators. If they include ads, AntennaPod plays them. We have nothing to do with those ads; all revenues go to whomever places the ads.
```

Maybe the hoster takes the money and the creator doesn't get anything.

Alternatively:

```suggestion
AntennaPod plays podcasts just like they were published by the creators. If they include ads, AntennaPod plays them. We have nothing to do with those ads, so all revenues go to the creators and/or their hosting company.
```